configure.ac

```diff
@@ -109,21 +120,31 @@ AS_IF([test "$GDBUS_CODEGEN" = ""],[
 	AC_MSG_ERROR([gdbus-codegen is required for build])
 ])
 
-AC_PATH_PROG([OSTREE],[ostree])
-AS_IF([test "$OSTREE" = ""],[
-	AC_MSG_ERROR([ostree is required for tests])
+AC_PATH_PROG([OSTREE_BINARY],[ostree])
+AS_IF([test "$OSTREE_BINARY" = ""],[
+	AC_MSG_ERROR([ostree command line tool is required for tests])
```

Comment:
When I added comments about this before, I just meant for you to change the error message, not s/`OSTREE`/`OSTREE_BINARY`/. I think that latter change makes things more confusing, given that the convention for program names in autotools is to be the program name in uppercase, i.e. `$(OSTREE)`, not `$(OSTREE_WITH_SOME_SUFFIX)`.

I merged an edited version of this commit to `master` already, which just changes the existing error messages, so you’ll get a minor rebase conflict on this. Could you drop the `_BINARY` changes at the same time please?